components/HarvesterServiceAddOnConfig.vue

```diff
@@ -125,6 +131,7 @@ export default {
           :mode="mode"
           :options="ipamOptions"
           :label="t('harvester.service.ipam.label')"
+          :disabled="mode === 'edit'"
```

Comment:
More of a question, but how come this component is disabled if the user is editing the form?

Reply:
![image](https://user-images.githubusercontent.com/18737885/149724142-aaa64c82-5777-4ee6-a79d-2c8a0b658821.png)

We can not change the ipam after LB is created.